code/modules/mob/living/carbon/human/emote_vr.dm

```diff
@@ -75,6 +75,8 @@
 				src.SpinAnimation(7,1)
 				message = "does a flip!"
 				m_type = 1
+		if ("vhelp") //Help for Virgo-specific emotes.
+			to_chat(src, "vwag, vflap, mlem, awoo, nya, peep, chirp, weh, merp, bark, his, squeak, nsay, nme, flip")
```

Comment:
`merp, bark, his, squeak`
One s? Might wanna fix that.

Reply:
The actual emote above is just `*his`

Reply:
Thats because the emotes delete the last 's' in emote regardless of what. So if you go *hiss, it'd play *his emote and if you go *his, it'd fail because '*hi' emote doesn't exist. So accurate command would be *hiss, at least in help menu

Reply:
Lmao, excellent. I'll make the adjustment when I'm off work later.

Classic jankiness

Reply:
Its less jankyness and more that system was made before any -s ending emotes were a thing. On other hand it allowed for things like *cough and *coughs or *salute and *salutes to be the same emotes.

Reply:
Classic features.